projects/kit/components/slider/slider.options.ts

```diff
@@ -15,13 +14,6 @@ export const TUI_SLIDER_DEFAULT_OPTIONS: TuiSliderOptions = {
 /**
  * Default parameters for Slider component
  */
-export const TUI_SLIDER_OPTIONS = new InjectionToken(
-    ngDevMode ? 'TUI_SLIDER_OPTIONS' : '',
-    {
-        factory: () => TUI_SLIDER_DEFAULT_OPTIONS,
-    },
+export const [TUI_SLIDER_OPTIONS, tuiSliderOptionsProvider] = tuiCreateOptions(
```

Comment:
Drop no move relevant code comment, please


```suggestion
export const [TUI_SLIDER_OPTIONS, tuiSliderOptionsProvider] = tuiCreateOptions(
```